projects/freerdp/build.sh

```diff
@@ -77,7 +60,7 @@ cmake_args=(
 
 # Build the project and fuzzers.
 rm -rf build
-cmake "${cmake_args[@]}" -S . -B build -G Ninja
+cmake "${cmake_args[@]}" -S . -B build -G Ninja -C ci/cmake-preloads/config-oss-fuzz.cmake
```

Comment:
```
reference to `__gxx_personality_v0'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `umtx_init':
(.text+0x3b): undefined reference to `std::condition_variable::condition_variable()'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `umtx_cleanup':
(.text+0xc0): undefined reference to `std::condition_variable::~condition_variable()'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `icu_66::umtx_initImplPostInit(icu_66::UInitOnce&)':
(.text+0x178): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: (.text+0x17f): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `icu_66::umtx_initImplPreInit(icu_66::UInitOnce&)':
(.text+0x25b): undefined reference to `std::condition_variable::wait(std::unique_lock<std::mutex>&)'
/usr/bin/ld: (.text+0x2be): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: (.text+0x2ca): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `icu_66::UMutex::getMutex()':
(.text+0x398): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `umtx_lock_66':
(.text+0x3ea): undefined reference to `std::__throw_system_error(int)'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `icu_66::umtx_initImplPostInit(icu_66::UInitOnce&)':
(.text+0x16e): undefined reference to `std::condition_variable::notify_all()'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libicuuc.a(umutex.ao): in function `std::call_once<void (&)()>(std::once_flag&, void (&)())::{lambda()#2}::_FUN()':
(.text._ZZSt9call_onceIRFvvEJEEvRSt9once_flagOT_DpOT0_ENUlvE0_4_FUNEv[_ZZSt9call_onceIRFvvEJEEvRSt9once_flagOT_DpOT0_ENUlvE0_4_FUNEv]+0x7): undefined reference to `std::__once_callable'
```

@akallabeth any ideas?

Reply:
@ligurio see above. you can do a test by adding `-DWITH_UNICODE_BUILTIN=ON`. if that works, we´ll add it to the ci script on our end.

Reply:
Fixed.